scss/application.scss

```diff
@@ -35,25 +35,16 @@
   z-index: 5;
 }
 
-.card-container{
-  display: flex;
-  flex-flow: row;
+.card-container {
+  display: grid;
+  grid-template-columns: repeat(3, max-content);
   align-items: center;
   max-width: 1200px;
   width: 100%;
```

Comment:
You are missing a `justify-content: center;` so the grid will be centered in the screen :smile: